kubernetes-1.24.yaml

```diff
@@ -167,7 +167,27 @@ subpackages:
         runs: |
           mkdir -p ${{targets.subpkgdir}}/usr/bin/
           CFLAGS="$CFLAGS -static -DVERSION=${{vars.pause-version}}-${{package.version}}"
-          gcc -o "${{targets.subpkgdir}}"/usr/bin/pause linux/pause.c
+          gcc -o "${{targets.subpkgdir}}"/usr/bin/pause-3.7 linux/pause.c
+
+  - name: kubernetes-1.24-default
```

Comment:
the `pause` binary is intentionally left out of the `-default` subpackage since it's not pinned to any specific kubernetes version, and typically only installed standalone